mapper/mapper.go

```diff
@@ -105,7 +106,12 @@ func CreateDataAggregationPage(cfg *config.Config, req *http.Request, basePage c
 func CreatePreviousReleasesPage(cfg *config.Config, req *http.Request, basePage coreModel.Page,
 	validatedQueryParams data.SearchURLParams, respC *searchModels.SearchResponse, lang string, homepageResponse zebedee.HomepageContent, errorMessage string,
 	navigationContent *topicModel.Navigation, template string, topic cache.Topic, validationErrs []coreModel.ErrorItem, zebedeeResp zebedee.PageData,
-) model.SearchPage {
+) (model.SearchPage, error) {
+	fmt.Printf("number of results: %d", respC.Count)
+	if respC.Count == 0 {
+		err := errors.New("no releases were found - there must be at least 1 release for a previous releases page to be rendered")
+		return model.SearchPage{}, err
+	}
```

Comment:
As referenced elsewhere, I don't think this is the right pattern.

Reply:
removed now